vindbaarheid/vindbaarheid.shacl.ttl

```diff
@@ -0,0 +1,492 @@
+@prefix premis: <http://www.loc.gov/premis/rdf/v3/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix meemoo: <http://data.meemoo.be/ns/>  .
+@prefix ebu: <http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix org: <http://www.w3.org/ns/org#> .
+
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+##############################
+#         NodeShapes         #
+##############################
+
+####################
+#      PREMIS      #
+####################
+premis:IntellectualEntity a rdfs:Class, sh:NodeShape ;
+    #? max en minCount constrains?
+    sh:property meemoo:identifierShape, meemoo:nameShape, meemoo:descriptionShape,
+    [
+        a sh:PropertyShape ;
+        sh:path premis:identifier ;
+        sh:class meemoo:LocalIdentifier ;
+        #? Aantal mogelijk?
+        sh:maxCount 2 ; #? Warning hierover? Meerdere PropertyShapes uiteen halen voor speciefiekere errors?
+        sh:name "lokale CP ID"@nl ;
+        sh:severity sh:Warning ;
+        sh:message "premis:identifier is not of class meemoo:LocalIdentifier"@en ;
+    ],
+    [
+        a sh:PropertyShape ;
+        sh:path schema:alternateName ;
+        sh:nodeKind sh:Literal ;
+        sh:datatype xsd:string ;
+        sh:name "alternate name"@en ;
+        sh:severity sh:Warning ;
+        sh:message "schema:alternateName is not of type string"@en ;
+    ],
+    [
+        a sh:PropertyShape ;
+        sh:path schema:partOfEpisode ;
+        sh:class schema:Episode ;
+        sh:maxCount 1 ;
+        sh:name "part of episode"@en ;
+        sh:severity sh:Warning ;
+        sh:message "schema:partOfEpisode is not of class schema:Episode"@en ;
+    ],
+    [
+        a sh:PropertyShape ;
+        sh:path schema:partOfSeries ;
+        sh:class schema:CreativeWorkSeries ;
+        sh:maxCount 1 ;
+        sh:name "part of series"@en ;
+        sh:severity sh:Warning ;
+        sh:message "schema:partOfEpisode is not of class schema:Episode"@en ;
+    ],
+    [
+        a sh:PropertyShape ;
+        sh:path schema:isPartOf ;
+        sh:class schema:ArchiveComponent ;
+        sh:maxCount 1 ;
+        sh:name "is part of"@en ;
+        sh:severity sh:Warning ;
+        sh:message "schema:isPartOf is not of class schema:ArchiveComponent"@en ;
+    ],
+    [
+        a sh:PropertyShape ;
+        sh:path schema:duration ;
+        sh:nodeKind sh:Literal ;
+        #? string in ISO 8601 of xsd:dateTime http://books.xmlschemata.org/relaxng/ch19-77049.html
+        sh:datatype xsd:string ;
+        sh:maxCount 1 ;
+        sh:name "duration"@en ;
+        sh:severity sh:Warning ;
+        sh:message "schema:duration is not of type string"@en ;
+    ],
+    [
+        a sh:PropertyShape ;
+        sh:path schema:dateCreated ;
+        sh:nodeKind sh:Literal ;
+        #? string of edtf:? (SPARQL quering & reasoning) https://periodo.github.io/edtf-ontology/
```

Comment:
EDTF, maar die ontologie is niet wat je nodig hebt. Hier zijn de namespaces: https://id.loc.gov/datatypes/edtf.html

Reply:
wij gebruiken http://id.loc.gov/datatypes/edtf/EDTF-level1